### PR TITLE
Update hosting providers

### PR DESCRIPTION
**Pull request by spacecodeit**
I have added signaltransmitter.de to the list of Hosting Providers.

### CODE REVIEW

**Comment by ohemorange** [comment]
Hey, thanks for submitting! I see that a line on the site says something like "you can activate a free Let's Encrypt certificate" -- that phrasing makes me think that maybe there's some button or checkbox the user has to click to turn on SSL for their website. Is that the case? If so, please add a link with instructions to do so to the "tutorial" field and change category to "partial" ("partial" means that the user has to take explicit action to turn on SSL).

**Comment by spacecodeit** [comment]
@ohemorange I have created and added a tutorial an changed the category

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -2298,5 +2298,15 @@
...
     "note": "Shared Web Hosting utilizes Plesk Panel interface to enable Let's Encrypt; Managed Cloud Hosting enabled via Certbot on request."
+  },
+  {
+    "name": "signaltransmitter.de",
+    "link": "https://signaltransmitter.de",
+    "category": "partial",
+    "tutorial": "https://signaltransmitter.de/faq/webhosting/lets-encrypt",
+    "announcement": "https://signaltransmitter.de/hosting",
+    "plan": "",
+    "reviewed": "",
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
    "reviewed": "2020.02.07",
```

Comment (spacecodeit) [reply _data/hosting_providers.json]: I have accepted your change.

**Pull request merged**